develop/schemadoc/Protocol/Protocol.Params.Param.Interprete.RawType.md

```diff
@@ -58,46 +56,36 @@ The following characters and digits:
 - e or E or ' ' (space)
 - a-f or A-F (if Protocol.Params.Param.Interprete.Base is 16)
 
-In the following example, the value -3 will be saved in database as 2D 33:
-
-```xml
-<RawType>numeric text</RawType>
-```
+The value -3 will be saved as string in the database as "2D33"
 
 ### only others
 
 Values made up of different types.
 
 See Protocol.Params.Param.Interprete.[Others](xref:Protocol.Params.Param.Interprete.Others).
 
-### other
-
-ASCII characters, both numeric and alphanumeric.
+> [!NOTE]
+> only others is depricated. It can still be used but is not recommended for new developments. We advice to use other in combination with [exception](xref:Protocol.Params.Param.Interprete.Exceptions.Exception) instead.
 
-Example:
+### other
 
-```xml
-<RawType>other</RawType>
-```
+any characters, both alphanumeric and non-alphanumeric.
 
 ### signed number
 
 When a next param is defined in Interprete.Type, the result will be the same as with unsigned number. When a fixed length is defined in Interprete.Type, the first half of the values will be positive number, and second half will be negative number.
 
+todo sentence below is confusing and incorrect when using different lengths
 Numbers between -128 and 127.
 
-In the following example, the value -3 will be saved in database as FDFFFFFF and the value 3 will be 03000000:
-
-```xml
-<RawType>signed number</RawType>
-```
+The value -3 will be saved in database as FDFFFFFF and the value 3 will be saved as 03000000:
```

Comment:
Perhaps specify how we arrive at these "weird" values, it's because the integer value is converted to a byte array before being converted into a hexadecimal notation. The byte array conversion is subject to the system's endianness, which is little-endian for x86-32 and x86-64 CPUs, which DataMiner is currently always running on.